com.unity.netcode.gameobjects/Runtime/Metrics/NetworkMetrics.cs

```diff
@@ -65,6 +65,8 @@ private static string GetSceneEventTypeName(uint typeCode)
         private readonly EventMetric<ServerLogEvent> m_ServerLogReceivedEvent = new EventMetric<ServerLogEvent>(NetworkMetricTypes.ServerLogReceived.Id);
         private readonly EventMetric<SceneEventMetric> m_SceneEventSentEvent = new EventMetric<SceneEventMetric>(NetworkMetricTypes.SceneEventSent.Id);
         private readonly EventMetric<SceneEventMetric> m_SceneEventReceivedEvent = new EventMetric<SceneEventMetric>(NetworkMetricTypes.SceneEventReceived.Id);
+        private readonly EventMetric<PacketEvent> m_PacketSentEvent = new EventMetric<PacketEvent>(NetworkMetricTypes.PacketEventSent.Id);
```

Comment:
I would make this plural as `m_PacketsSentEvent`, to clarify that it's not one event per packet but rather one event to record multiple packets

Reply:
I might rename `PacketEvent` to `PacketCountEvent` for the same reason 🤔

Reply:
Every Event we have are not more precise in term of name. We've discussed also on slack adding the byte count to the packet event. Might also be useful for packet loss or RTT.

As for the plural, we don't have plural for other event, that can also be multiple. For consistency, I'll keep things as is.

Reply:
Because I have total forgot about the fact we have counters as well as event (since we used mostly just events). However, if we want to track packet overhead, event would be more appropriate no?

Reply:
Even in that case, I think two separate counters would be appropriate rather than events. I think events would make sense if we had a use case where we need to differentiate packets from each other. But I don't know if we have that use case. By putting it in events, I think we make tracking total byte count harder, because we have to sum that information out of the packet events

Reply:
Makes sense. The only reason I can see why if we want to track packet per connection which Event allow us. In the meantime, I'll move this to counters and we can always move back to Events if we need.